### DISCUSSION TITLE
Problems queryRaw

### DISCUSSION THREAD

**Discussion by orcioly** [prisma/prisma#20513] [Q&A]
### Question

I'm developing an API where the database is MySQL and has no relationships and in this case I'm using Prisma's queryRaw.
I tested the code below directly in the bank and it works 100% but in the code it doesn't work and doesn't show errors, Insomnia just keeps processing and doesn't finish. I believe that the JSON_OBJECT is not being recognized in the code, but it works directly in the database running this script.

I put it in a try catch for testing and the console gives an error message: "FUNCTION does not exist in the bank". But if I run the same code directly on the bank it works without problems.

Follow the code below:

```
const voucher = await prismaClient.$queryRaw<TrackingVoucher>`
  SELECT 
      vt.id,
      vt.transactionId,
      vt.referenceId,
      vt.status_venda,
      vt.description, vt.name,
      tv.email_comprador,
  JSON_OBJECT(
      'id', te.id,
      'nome_evento', te.nome_evento
  ) AS event
  FROM
    venda_transacoes AS vt
  JOIN
    tracking_vouchers AS tv ON tv.superpnr = vt.referenceId
  INNER JOIN
    tracking_eventos AS te ON te.id = tv.id_evento
  WHERE
    vt.referenceId = ${input.superpnr} AND tv.email_comprador = ${input.email_comprador}
`;
```
Could someone help me?

**Comment by ludralph** [comment]
Closing because it's a duplicate of [#20511](https://github.com/prisma/prisma/discussions/20511)
